cs-utilities/src/main/java/io/cloudslang/content/utilities/actions/FindTextInPdf.java

```diff
@@ -0,0 +1,95 @@
+import static org.apache.commons.lang3.StringUtils.countMatches;
+import static org.apache.commons.lang3.StringUtils.defaultIfEmpty;
+
+/**
+ * Created by marisca on 7/11/2017.
+ */
+public class FindTextInPdf {
+
+    /**
+     * This operation checks if a text input is found in a PDF file.
+     *
+     * @param text       The text to be searched for in the PDF file.
+     * @param ignoreCase Whether to ignore if characters of the text are lowercase or uppercase.
+     *                   Valid values: "true", "false". For any other value the ignoreCase will be set to "false".
+     *                   Default Value: "false"
+     * @param pathToFile The full path to the PDF file.
+     * @param password   The password for the PDF file.
+     * @return - a map containing the output of the operation. Keys present in the map are:
+     * returnResult - The number of occurrences of the text in the PDF file.
+     * returnCode - the return code of the operation. 0 if the operation goes to success, -1 if the operation goes to failure.
+     * exception - the exception message if the operation fails.
+     */
+
+    @Action(name = "Find Text in PDF",
+            description = FIND_TEXT_IN_PDF_OPERATION_DESC,
+            outputs = {
+                    @Output(value = RETURN_CODE, description = RETURN_CODE_DESC),
+                    @Output(value = RETURN_RESULT, description = FIND_TEXT_IN_PDF_RETURN_RESULT_DESC),
+                    @Output(value = EXCEPTION, description = EXCEPTION_DESC),
+            },
+            responses = {
+                    @Response(text = SUCCESS, field = RETURN_CODE, value = ReturnCodes.SUCCESS, matchType = COMPARE_EQUAL, responseType = RESOLVED, description = SUCCESS_DESC),
+                    @Response(text = FAILURE, field = RETURN_CODE, value = ReturnCodes.FAILURE, matchType = COMPARE_EQUAL, responseType = ERROR, isOnFail = true, description = FAILURE_DESC)
+            })
+    public Map<String, String> execute(
+            @Param(value = TEXT, required = true, description = INITIAL_VALUE_DESC) String text,
+            @Param(value = IGNORE_CASE, description = IGNORE_CASE_DESC) String ignoreCase,
+            @Param(value = PATH_TO_FILE, required = true, description = DEFAULT_VALUE_DESC) String pathToFile,
+            @Param(value = PASSWORD, description = PASSWORD_DESC, encrypted = true) String password) {
+
+        try {
+            final Path path = Paths.get(pathToFile);
+            final String pdfPassword = defaultIfEmpty(password, "");
+            final String pdfContent = PdfParseService.getPdfContent(path, pdfPassword).trim().replace(System.lineSeparator(), "");
+            final boolean validIgnoreCase = toBoolean(defaultIfEmpty(ignoreCase, FALSE));
+
+            if (validIgnoreCase)
+                return getSuccessResultsMap(valueOf(countMatches(pdfContent.toLowerCase(), text.toLowerCase())));
+
+            return getSuccessResultsMap(valueOf(countMatches(pdfContent, text)));
```

Comment:
possible duplication of code? `valueOf(countMatches(pdfContent...., text....)` maybe extract the if in a method with 3 params `(pdfContent, text, validIgnoreCase)` and call `getSuccessResultsMap` on the result